Langs/040D.lang

```diff
@@ -39,10 +39,10 @@ Supported=נתמך
 Not Supported=לא נתמך
 Disabled / Not Detected=מושבת / לא זוהה
 Drive(s) Meet Requirements=הכוננ/ים עונה/ים על הדרישות
-TPM Missing / Disabled=TPM לא קיים / מושבת
```

Comment:
TPM Missing / Disabled=TPM לא קיים / מושבת

Reply:
This starts with English and continues with Hebrew, so the sentence will shown from left to right.
That's why I reverted this change...